core/src/test/scala/org/apache/spark/scheduler/EventLoggingListenerSuite.scala

```diff
@@ -163,6 +163,7 @@ class EventLoggingListenerSuite extends SparkFunSuite with LocalSparkContext wit
   test("SPARK-31764: isBarrier should be logged in event log") {
     val conf = new SparkConf()
     conf.set(EVENT_LOG_ENABLED, true)
+    conf.set(EVENT_LOG_COMPRESS, false)
```

Comment:
Hmm, only this test? Other tests in the suite?

Reply:
If `EVENT_LOG_COMPRESS` is true, will this test fail?

Reply:
Yes, this is the only failure instance as of now, @viirya .